Taskfile.yml

```diff
@@ -32,10 +32,15 @@ tasks:
         vars: {STAGE_NAME: "dev", IMAGE_NAME: "{{.DEV_IMAGE_NAME}}:{{.IMAGE_VERSION}}"}
 
   run-lint:
-    desc: "Run ruff check and format"
+    desc: "Run ruff"
     cmds:
-      - ruff check --fix
-      - ruff format
+      - cmd: ruff check --fix
+        ignore_error: true
+      - pyupgrade `find . -name "*.py"`
```

Comment:
Add pyupgrade to dev dependencies group.

See - https://github.com/saurabheights/Pixion/blob/54310bf5a259fee4d5e2500bbced502480788033/pyproject.toml#L41